.github/workflows/release-please.yml

```diff
@@ -14,7 +14,6 @@ jobs:
       - uses: google-github-actions/release-please-action@v3
         id: release
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Without the token, CI is not run on the release pull requests.

Reply:
Not according to their docs: https://github.com/google-github-actions/release-please-action#setting-up-this-action

Reply:
It's not specific to this action, see https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow.
But you can disregard my comment because there's no point passing `secrets.GITHUB_TOKEN` (we need a PAT to fix the issue). We can see in https://github.com/nodejs/node-core-utils/pull/630 that no CI was run.

Reply:
Ah gotcha, yes passing the token won't help with that.